test/module/cfn_json/test_cfn_json.py

```diff
@@ -47,7 +47,7 @@ def setUp(self):
             },
             "vpc_management": {
                 "filename": 'test/fixtures/templates/quickstart/vpc-management.json',
-                "failures": 31
+                "failures": 35
```

Comment:
Again because adding a check for `DestinationCidrBlock`.  The actual number after deduplication is less than 4 but this check runs before  deduplication.